publishTag

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+BUCKET=codesplain-ui
```

Comment:
As far as I can tell this is the only difference between these two scripts. Make it a flag instead of 2 scripts.